SPECS/rsyslog/rsyslog.logrotate

```diff
@@ -0,0 +1,40 @@
+/var/log/syslog
+{
+    rotate 7
+    daily
+    missingok
+    notifempty
+    delaycompress
+    compress
+    postrotate
+        /usr/bin/systemctl kill -s HUP rsyslog.service >/dev/null 2>&1 || true
+    endscript
+}
+
+ 
+
+/var/log/mail.info
+/var/log/mail.warn
+/var/log/mail.err
+/var/log/mail.log
+/var/log/daemon.log
+/var/log/kern.log
+/var/log/auth.log
+/var/log/user.log
+/var/log/lpr.log
+/var/log/cron.log
+/var/log/debug
+/var/log/messages
+{
+    rotate 4
+    weekly
+    size 50M
```

Comment:
From your logs and [this online discussion](https://serverfault.com/questions/391538/logrotate-daily-and-size) it seems like `weekly` is going to be ignored when `size` is used. Since our version of `logrotate` is > 3.8.1, [we should be able to use `maxsize`](https://serverfault.com/questions/474941/how-to-rotate-log-based-on-an-interval-unless-log-exceeds-a-certain-size) instead to take both settings into account. See also: [`logrotate` man pages](https://man7.org/linux/man-pages/man8/logrotate.8.html).

```suggestion
    weekly
    maxsize 50M
```